datawrapper/datawrapper
# Fix upload textarea placeholder not visible (v2)

New attempt to do #422 but this time don't break tab-separated data that starts with an empty column.

#### Testing

1. Paste this in the data textarea.

    ```
    	foo	1.0
    2	bar	2.0
    3	baz	3.0
    ```

2. Refresh the page.

3. See that the data in the textarea is unchanged (no whitespace trimming).
